src/accordion/accordion-item.tsx

```diff
@@ -25,6 +31,22 @@ function _AccordionItem(
     }: AccordionItemProps,
     ref: React.Ref<HTMLDivElement>
 ) {
+    useImperativeHandle(
+        ref,
+        () => {
+            if (!isFunction(ref)) {
```

Comment:
this check here would mean ref callbacks won't work though

if the underlying div element isn't needed, then this will be sufficient

```
    useImperativeHandle(
        ref,
        () => ({
            expand(): void {
                setExpand(true);
            },
            collapse(): void {
                setExpand(false);
            },
        }),
        []
    );
```

if you do need the reference to the div, tracking an internal element ref could work
```
    const elementRef = useRef<HTMLDivElement>(); // this is passed to the container
    useImperativeHandle(ref, () => {
        return {
            ...elementRef.current,
            expand(): void {
                setExpand(true);
            },
            collapse(): void {
                setExpand(false);
            },
        };
    });
```

Reply:
need the ref to div, had to use Object.assign instead of the spread operator